hack/rpm-deps.sh

```diff
@@ -219,6 +219,7 @@ handlerbase_extra="
 bazel run \
     --config=${ARCHITECTURE} \
     //:bazeldnf -- rpmtree --public --arch=aarch64 --name handlerbase_aarch64 \
+    --basesystem centos-stream-release \
```

Comment:
This is also in the wrong commit - it belongs to "rpm: Switch to CentOS Stream 8", where all other ontainers are switched from Fedora to CentOS Stream.

Reply:
Fixed.